### PR TITLE
data: omlondhe

### PR DESCRIPTION
**Pull request by omlondhe**
Added my profile

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6283"></a>

### CODE REVIEW

File: data/omlondhe.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+    "name": "Om Prashant Londhe",
+    "displayStatsPublic": true,
```
Comment (khushalarora) [inline data/omlondhe.json:3]: You can remove this line, it is not required anymore :+1: 
```Suggestion
```

File: data/omlondhe.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
...yrIGtFj+Me8RkKRI3DfyrNqZIrRgi4p2RzyUqoMA0FOvZ6yWLwSD4QIjcz10pORpTvgNxFRs10IzHQeXqOpp8t8WkTxVyVmiSygfIQSTrJ9dCfQ1C/hDJh+4AjTsSd6JtYy02uZCezDaN586ovZ2M+FhsBsfnNeYozTtnoKn0K7+EuEkNlYDYuJPpEaUInD0DjOiCNQRmAPYiSKasr5j4fDHI6z84IqItuxjxR5CT8zVeUg0i/3z9YnY6GgLXB87gm8+aZzNBiO9FIjDwgc9STtV5SDNLykvJzimOP7LH+bc+Y/avKXfxj/mPyr2m+0T6zFJdQHRo+YohED8wfPWlrrr2qr6eVaDmvY5GFI+Esv+0yPkauuY+6NJU8jupNLLWNuLzzdm/erBxE6ynyNcK4h2G4iVBVwwG87ifSibHGyQFXKVB2aJ9egpImJVm8Xh6cvrRItK06Bo6j9RrXXTBxbVI11niZQykpMRlkiO4NQPFTfzRuNx3G5FZX3twKApYKDsdR86Z8LuZXzZfiiY112NdbO4pD3itsXsMQPiiR0kcqyGFusCEYAes89u1bnCqGtsvLWsAoJuEDfMfvv5CgVxun2MXQu6QCdTt2pPxm/LsP9RJ+ZrU8Kt5S51I/CYiAdfUk/asdidXefzN9zXRSsX5Er8dgsVWd6OW2p5x51RjLOUqZBnSrRezFJaOt0UiEAUIhpjhzn23A/lVH0JHsFupBqCimT4UkbGhTaPSpNlUj2JFL7a60zsryNAqBJjaaaDsSaCDtUglRTXSixboydAggcLVmXSrcletb0qRVIpw+KdTGdx/7H96cWMe8fG3rr96TtaovDGlaTGTYxPEbinUgjy/anmAdLyMRmVwCQJBBMacqzbpNR/jHs+JTquw6noaXjH0HlL2R/t9/8r6n9q6qv/KL/S3/AMB+9e0eEQ8pAD1S1dXUTRLsuO48qhXV1cii6I3NqL4T8DV1dXMj/WOMJ/h+le8L2X+vxV7XUECfZouD7N6/c1isH/i3vM//AGV1dXeGdHs1dv8Aw/UfasDiPjb/AHH7mva6hDs75BAUPifiHl+1dXVaPZkl0dTLgnxN/t/UV1dVJdE49j1aGfnXV1RZVAuJ2PlSa1tXtdT4hMgRhtxTAV1dXZA4+jhXrV1dUypGp2dzXtdSnBQoLinwjzFdXUTl2LK6urq4qf/Z",
```
Comment (khushalarora) [inline data/omlondhe.json:6]: You can remove this avatar line, it is not required anymore :+1: 
```Suggestion
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @omlondhe, thank you for contributing :tada:

**Comment by omlondhe** [comment]
Hey, @khushalarora Thank you for replying, changes are done.

**Comment by omlondhe** [comment]
By the way, I'm adding a feature to the project. You will receive a pull request in the next 5 minutes, kindly review it.

**Comment by omlondhe** [comment]
Hey @khushalarora , done with changes for this page.

The feature that I was talking about is also added in PR, take a look.
Along with that, I have also raised an issue in the production code that I have solved locally, will send you a PR once you merge both of these PRs so the issue will be merged separately.

Thank you!

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**